### PR TITLE
K8s update

### PR DESCRIPTION
**Pull request by deeghuge**
mark stage operation as not implemented

### CODE REVIEW

**Review by hemalathagajendran — APPROVED** [review]

File: driver/csiplugin/gpfs.go [hunk 1]
```diff
@@ -210,6 +210,8 @@ func (driver *ScaleDriver) SetupScaleDriver(ctx context.Context, name, vendorVer
 	// Adding Capabilities
 	vcam := []csi.VolumeCapability_AccessMode_Mode{
 		csi.VolumeCapability_AccessMode_MULTI_NODE_MULTI_WRITER,
+		csi.VolumeCapability_AccessMode_SINGLE_NODE_MULTI_WRITER,
+		csi.VolumeCapability_AccessMode_SINGLE_NODE_SINGLE_WRITER,
```
Comment (Jainbrt) [inline driver/csiplugin/gpfs.go:214]: should we add SINGLE_NODE_WRITER volumecapability too for backward compatibility SINGLE_NODE_MULTI_WRITER & SINGLE_NODE_SINGLE_WRITER ?

Comment (deeghuge) [reply driver/csiplugin/gpfs.go:214]: Done

Comment (Jainbrt) [reply driver/csiplugin/gpfs.go:214]: do we also require to add similiar capability in NodeserviceCapability ?

Comment (deeghuge) [reply driver/csiplugin/gpfs.go:214]: no

**Review by Jainbrt — APPROVED** [review]

**Pull request merged**